fanuc_cr7ia_support/package.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>fanuc_cr7ia_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the Fanuc CR-7iA (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Fanuc CR-7iA manipulators. This currently includes the /L model
+      only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>CR-7iA/L - Normal Range</li>
```

Comment:
re: limits based on J2-J3: true, but in *normal range* only refers to the range of motion of J1 in this case.